lib/ci-helper.ts

```diff
@@ -274,6 +277,7 @@ export class CIHelper {
 
         if (optionsUpdated) {
             await this.notes.set("", options, true);
+            await this.notes.push(this.urlRepo);
```

Comment:
`CIHelper::handlePR`  is doing a push.  Is any of this necessary now?

Reply:
Good point! I'll drop this call.